mlem/contrib/kubernetes/base.py

```diff
@@ -0,0 +1,126 @@
+import os
+from typing import ClassVar, Optional
+
+from kubernetes import config
+
+from mlem.config import project_config
+from mlem.core.objects import (
+    DeployState,
+    DeployStatus,
+    MlemDeployment,
+    MlemEnv,
+    MlemModel,
+)
+from mlem.runtime.server import Server
+from mlem.ui import EMOJI_BUILD, EMOJI_OK, echo, set_offset
+
+from ..docker.base import (
+    DockerEnv,
+    DockerImage,
+    DockerImageBuilder,
+    DockerRegistry,
+    generate_docker_container_name,
+)
+from .context import K8sYamlBuildArgs, K8sYamlGenerator
+
+POD_STATE_MAPPING = {
+    "Pending": DeployStatus.STARTING,
+    "Running": DeployStatus.RUNNING,
+    "Succeeded": DeployStatus.STOPPED,
+    "Failed": DeployStatus.CRASHED,
+    "Unknown": DeployStatus.UNKNOWN,
+}
+
+
+class K8sDeploymentState(DeployState):
+    type: ClassVar = "kubernetes"
+    image: Optional[DockerImage]
```

Comment:
set default values to None to be able to create empty state

Reply:
done